presto-elasticsearch/pom.xml

```diff
@@ -367,6 +367,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Are we trying to maintain a specific order of dependencies in `pom.xml`?

Reply:
"We" (I try to) do. Order by scope, group id, alphabetically (group id, artifact id). Scope order is compile, provided, test. Group id order is presto, airlift, rest.

Reply:
i don't think there is any specific order in these files.

i tried to introduce in some, but without clear written rules, this is descending into chaos quickly